src/ipc/ipc3/dai.c

```diff
@@ -57,6 +57,20 @@ int dai_config_dma_channel(struct comp_dev *dev, void *spec_config)
 					      dd->stream_id);
 		channel = EDMA_HS_GET_CHAN(handshake);
 		break;
+	case SOF_DAI_AMD_BT:
+		// Select if it is TX or RX channel
+		channel = dai_get_handshake(dd->dai, dai->direction,
+					    dd->stream_id);
+		break;
+	case SOF_DAI_AMD_SP:
+		// Select if it is TX or RX channel
```

Comment:
I don't think we use `//` comments in common code

Reply:
Addressed the Review Comment. Done